tools/windows-node-installer/pkg/types/types.go

```diff
@@ -4,6 +4,13 @@ package types
 // sub-package
 // TODO: Move every cloud provider types here
 
+const (
+	// KubeletPort no that will be opened on the windows node via firewall
+	KubeletPort = "10250"
```

Comment:
This is not a kubelet port but rather pod needed for container logs. Please rename to `ContainerLogsPort`.

Reply:
Please address this comment.